.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "lib/forge-std"]
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/openzeppelin/openzeppelin-contracts
```

Comment:
I think that to pin it to 4.7.3 we need to install
`https://github.com/openzeppelin/openzeppelin-contracts@v4.7.3`